platform/viewer/src/index.js

```diff
@@ -34,11 +36,21 @@ const appProps = {
     OHIFMeasurementTrackingExtension,
     OHIFDICOMSRExtension,
     OHIFDICOMVIDEOExtension,
+    OHIFDICOMPDFExtension,
   ],
 };
 
-/** Create App */
-const app = React.createElement(App, appProps, null);
-
-/** Render */
-ReactDOM.render(app, document.getElementById('root'));
+const { defaultTheme="theme" } = appProps.config;
+/**
+ * Load the default theme settings,
+ * and then render the app in the then block so that themes get
+ * loaded early enough to modify the initial render.
+ */
+loadSearchConfigPoint(defaultTheme, '/theme', 'theme').then(() => {
```

Comment:
tagging @JamesAPetts for core

Reply:
This is only changing the location of existing code to allow upgrading the version, but config-point is already in v3-stable.

Reply:
I know @swederik dind't like the name theme?

So loadSearchConfigPoint comes from config-point, I thought we were removing it as a dependency of this PR? I get it was already in v3 stable, but aren't we now using it?

Now we decided v32-stable is the home of `config-point` for now, can we now remove `config-point` entirely from v3-stable?

Reply:
@wayfarer3130 This is the only comment left, can we please address this and merge this outstanding branch?